ops/storage-ddls/cassandra-setup.cql

```diff
@@ -231,8 +248,8 @@ CREATE TABLE events(
     topics frozen<set<text>>,
     placeids frozen<set<text>>,
     fulltext text, /* conjunction of title and body to enable querying of both at the same time */
-    PRIMARY KEY (eventid)
-);
+    PRIMARY KEY (eventid, eventtime)
+) WITH CLUSTERING ORDER BY (eventtime DESC);
```

Comment:
Not sure that this will do what we want it to. This looks a lot like one of the anti-patterns mentioned in the [Cassandra docs](https://www.datastax.com/dev/blog/we-shall-have-order), namely the "postsbyuser" example. Are you sure that this is the best way to do the sorting? Sorting in Cassandra is hard :(